aha-table.html

```diff
@@ -21,14 +21,14 @@
     <content id="columndefs"></content>
 
     <template is="dom-if" if="{{showColumnChooser}}">
-      <div class="flex columnChooser u-mb-">
+      <div class$="{{_getColumnChooserClass(greedyHeightWithScroll)}}">
```

Comment:
class$ is a dynamic class attribute, which can not contain static and dynamic class name. There are a lot of example of this pattern in the px-data-table code. let me know if you like me to point them to you.